mods/cnc/languages/rules/en.ftl

```diff
@@ -489,8 +486,8 @@ actor-bio-husk-name = Biological Lab (Destroyed)
 
 actor-hosp =
    .name = Hospital
-   .captured-desc = Provides infantry with self-healing.
-   .capturable-desc = Capture to enable self-healing for infantry.
+   .captured-desc = Infantry can self-heal.
+   .capturable-desc = Once captured, infantry can self-heal.
```

Comment:
This makes the wording "Capture to X" against all the `capturable-desc` that was previously consistent now inconsistent. I think we should keep a consistent pattern of wording here, whether that's the original or a new one I don't mind.